packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -1406,6 +1406,60 @@ You're able to grant third-party admin status when you assign a new role, or you
 
 When this setting is enabled, the admins won't receive any of the default Okta administrator emails. These admins also won't have access to contact Okta Support and open support cases on behalf of your org.
 
+### List Users with Role Assignments
+
+<ApiOperation method="get" url="/api/v1/iam/assignees/users" />
+
+Lists all the Users with Role Assignments
+
+##### Request parameters
+
+
+| Parameter     | Description                                          | Param Type  | DataType  | Required  |
+| :------------ | :--------------------------------------------------- | :---------- | :-------- | :-------- |
+| `after`       | Specifies the pagination cursor for the next page of targets   | Query        | String     | FALSE    |
+| `limit`       | Specifies the number of results for a page (default is 100)     | Query        | Number     | FALSE    |
+
+##### Response parameters
+
+Array of RoleAssignedUser
```

Comment:
should this be a link?

Reply:
@vasishtajayanti-okta Is there an object that we should link to?

Reply:
@brentschaus-okta I haven't created a separate object here as the response is pretty simple.
Let me know if you recommend creating a separate object here.

This is what the object would look like:
```
      RoleAssignedUser:
      type: object
      properties:
        id:
          type: string
          readOnly: true
        orn:
          type: string
          readOnly: true
        _links:
          $ref: '#/components/schemas/LinksSelfAndRoles'
```
Ex.
```
    {
      "id": "00u118oQYT4TBGuay0g4",
      "orn": "orn:okta:00o5rb5mt2H3d1TJd0h7:users:00u118oQYT4TBGuay0g4",
      "_links": {
        "self": {
          "href": "http://your-subdomain.okta.com/api/v1/users/00u118oQYT4TBGuay0g4"
        },
        "roles": {
          "href": "http://your-subdomain.okta.com/api/v1/users/00u118oQYT4TBGuay0g4/roles"
        }
      }
```

Reply:
@vasishtajayanti-okta Sounds good :)